store/sqlstore/team_store.go

```diff
@@ -1173,6 +1176,7 @@ func (s SqlTeamStore) GetTeamsForUserWithPagination(userId string, page, perPage
 	return dbMembers.ToModel(), nil
 }
 
+// GetChannelUnreadsForAllTeams returns all the messages that are unread in the channel for all teams.
```

Comment:
This is not accurate, it doesn't return the messages, it returns the information about unreads/mentions/notifyProps for all the channels in all the teams (except the excluded team).

Reply:
fixed it